### ISSUE TITLE
Can't publish scoped package

### ISSUE THREAD

**Issue by drazik** [semantic-release/semantic-release#512]
Hello,

First, thank you for you work. This tool is amazing.

But I have a little problem to publish a package under my company scope. I setup everything, then push a commit to run the publish process on Travis CI. The job completes, and the `semantic-release` npm script is started, but I get the following error : 

```
npm ERR! publish Failed PUT 402
npm ERR! code E402
npm ERR! You must sign up for private packages : @wandiparis/troposphere
```

My user is in the wandiparis org on npm, and when I publish other packages manually I don't have any problems.

I am wondering how semantic-release know who I am on npm when running in Travis CI. I entered my npm login and password during `semantic-release-cli setup`, is it send to Travis CI or something ?

Did somebody already have this problem and find a solution ?

Thank you very much.

**Comment by pvdlg** [comment]
Authentication is required to publish scoped packaged.
The authentication is set up with the `NPM_TOKEN` environment variable.

Can you check that `NPM_TOKEN` is set on your CI environment (for example in [Travis](https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings)) and the token set allow you to publish?

**Comment by drazik** [comment]
There was already a `NPM_TOKEN` environment variable in the configuration on my repository in Travis CI. I tried to remove it, create a new token using `npm token create`, according to [npm documentation](https://docs.npmjs.com/getting-started/working_with_tokens#how-to-create-a-new-full-permission-token) and adding it again, but the problem was still the same.

I found that the [npm documentation](https://docs.npmjs.com/getting-started/scoped-packages#publishing-a-scoped-package) says that I need to set the `--access=public` option in the `npm publish` command to be able to publish the package. I updated my npm `semantic-release` script to the following : 

```
"semantic-release": "semantic-release pre && npm publish --access=public && semantic-release post"
```

After that, I got another issue :

```
semantic-release ERR! pre This test run is not the build leader and therefore a new version won’t be published.
```

This seems to be related to https://github.com/semantic-release/semantic-release/issues/390. The error message says that "a new version won't be published" but a new version has been released.

To sum up, my issue was not about `NPM_TOKEN`, because semantic-release adds it to the Tracis CI repository settings, but about `--access=public` in the `npm publish` command, because I want to publish a scoped package.

Thanks for your help @pvdlg.

**Comment by priestine** [comment]
In case you have this problem, what helped me out was that I added
```
"publishConfig": {
    "access": "public"
  }
```
to my `package.json`. As for **semantic_release** script, I used
```
...
"semantic-release": "semantic-release"
```

Works like a charm now.

**Comment by paulocoghi** [comment]
A simpler solutions is to use:

    npm publish --access public
